src/cascade/dismod/db/wrapper.py

```diff
@@ -267,10 +279,9 @@ def _check_column_types_actually_written(self):
             for table_name, table_definition in self._table_definitions.items():
                 introspect = text(f"PRAGMA table_info([{table_name}]);")
                 results = connection.execute(introspect)
-                if results.returns_rows:
-                    table_info = results.fetchall()
-                else:
-                    continue  # Not all tables are in all databases.
+                if not results.returns_rows:
+                    continue
+                table_info = results.fetchall()
```

Comment:
I have heard that Code Complete likes if/else.  
I think you're saying, no else is ok.

Reply:
What would the else add when there's an explicit exit in the if? Code Complete seems to be confused about a lot of things.